src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -213,6 +213,8 @@ private boolean joinEvent(HttpServletRequest request, HttpServletResponse respon
     
     try {
       UserStorage.joinEvent(currentUserId, eventId);
+      User user = UserStorage.getUser(currentUserId);
+      Utils.joinGCalendarEvent(currentUserId, eventId, user.getEmail());
```

Comment:
The currentUserId needs to be changed with the owner of the event id

Reply:
oops)